### DISCUSSION TITLE
Is the render component open source

### DISCUSSION THREAD

**Discussion by jk** [Fechin/reference#134] [Q&A]
Hi,

I like your cheatsheets a lot. I would like to create my own cheatsheets for my own small scripts I have laying around here, so I wonder if the component which takes the reference repo to generate the HTML pages is open source, so I can host it on my internal raspberry pi?

**Comment by srikrishnacj** [comment]
I do really like this very clean interface of this site. I would like to host my own cheatsheets specific to my daily work. It would be great if you can make this site as opensource.
